frontend/src/components/admin/UsersTable.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import { Icon } from "@chakra-ui/icon";
+import { MdDelete } from "react-icons/md";
+import {
+  Badge,
+  Link,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  IconButton,
+} from "@chakra-ui/react";
+import { User } from "../../APIClients/queries/UserQueries";
+import convertLanguageTitleCase from "../../utils/LanguageUtils";
+
+export type UsersTableProps = {
+  isTranslators?: boolean;
```

Comment:
nit: `isTranslators` sounds weird to me, but i'm not sure if `areTranslators` would be any better
commenting here in case anyone has suggestions for a better name (but also fine with leaving it)